app/src/main/res/values/strings.xml

```diff
@@ -209,8 +209,8 @@
     <string name="remember_answers_warning_title">Notice!</string>
     <string name="remember_answers_warning_desc">It is important to remember your answers. You will be required to provide exactly the same answers when restoring your account and will not be able to restore your account without them. The only way to reset your answers is by creating a new backup.</string>
     <string name="dialog_continue">I understand, continue</string>
-    <string name="send_feedback_title">Feedback</string>
-    <string name="send_feedback_text">We can’t respond to feedbacks individually.\nIf you have a question or need help resolving a problem, you’ll find answers in our help center.</string>
+    <string name="send_feedback_title">How can we do better?</string>
+    <string name="send_feedback_text">Thoughts? Suggestions? Share them here! If you need help resolving a problem, check out ״I Need Help"</string>
```

Comment:
there are two different types of double quotes starting and finishing ״I Need Help"